### ISSUE TITLE
Slave Pens and Talbuk Mastery

### ISSUE THREAD

**Issue by bm3737** [dalaranwow/dalaran-wow#4841]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Slaves in the slavepens will come out of their cages. They are not able to be targeted and cannot be killed.
When attempting to kill Bach'lor with my abilities it says "invalid target". I can use melee attacks and aoe abilities and that is all.

**Expected behaviour**: They should stay in their cages and not agro onto whoever walks near them. I should be able to use my abilities on him.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by dreamingdust** [comment]
I am pretty certain that you needed to just click Alt to fix the invalid target issue with Bach'lor. Sometimes if you alt + tab the game considers alt as clicked after you go back to the game. And unless you change the bind Alt + Spell is self-cast, which, considering you are trying to use a DMG ability, is impossible to be done, because as the game says, you are an invalid target, not Bach'lor.

**Comment by TeridanDW** [comment]
I can't confirm any of the 2 issues posted above. Please provide a steps to reproduce the problems and i will reopen this report.
Thank you.
